examples/notebook/tensorflow/vgg19_ibean/inc_quantize_model.py

```diff
@@ -27,13 +31,14 @@
 
 def preprocess(image, label):
     image = tf.cast(image, tf.float32)/255.0
-    return tf.image.resize(image, [w, h]), tf.one_hot(label, class_num)
+    return  tf.image.resize(image, [w, h]), np.array(tf.one_hot(label, class_num))    
```

Comment:
np.array() return numpy array.
tf.one_hot return tf.tensor.
preprocess() should return (tf.tensor, tf.tensor).
please rm np.array() if code is passed.

Reply:
WIth INC 2 TF dataloader, if we use tf.tensor for labels, it causes that labels and preds type not matching issue. 
we can directly use labels. Let me change it.

Reply:
OK! ignore my comment